### DISCUSSION TITLE
The main window

### DISCUSSION THREAD

**Discussion by davidfurdos** [Buduf/PTI-Ear-Trainer#7] [Ideas]
I have made a preview about the main menu of the program. Please let me know if you agree with it.
![Main menu](https://user-images.githubusercontent.com/108064994/196930606-403b1dc3-ebf0-4747-8db0-f403e4d5b704.jpeg)

**Comment by Buduf** [comment]
I think it looks good. But I think casual and practice mode will be the same.

Also we need a control for selecting the music instrument. The selected instrument's sound will be played on the game window. We will have a handful of instruments, so maybe a mini slideshow with the names and pictures of the instruments can be created. If no such control exists, we might have to create a custom control.

**Reply by Andre2220** [reply]
I also think it looks cool.
